modules/nw-kube-proxy-about.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+// * networking/configuring-kubeproxy.adoc
+
+[id="about-kubeproxy-{context}"]
+= About kube-proxy
+
+The Kubernetes network proxy (kube-proxy) runs on each node. It maintains
```

Comment:
Would you need to reuse this module? I think it might be fair to put it in the assembly itself.